xcm/src/v1/junction.rs

```diff
@@ -41,10 +41,7 @@ pub enum BodyId {
 	Named(Vec<u8>),
 	/// An indexed body.
 	// TODO: parity-scale-codec#262: Change to be a tuple.
```

Comment:
https://github.com/paritytech/parity-scale-codec/issues/262

This is completed